vayesta/core/types/wf/fci.py

```diff
@@ -450,3 +450,12 @@ def as_cisdtq(self, c0=None):
         #c3 = (c3aaa, ...
         #c4 = (c4aaaa, ...
         return wf_types.UCISDTQ_WaveFunction(self.mo, c0, c1, c2, c3, c4, projector=self.projector)
+
+    def as_ccsd(self):
+        return self.as_cisd().as_ccsd()
+
+    def as_ccsdtq(self):
+        return self.as_cisdtq().as_ccsdtq()
+
+    def as_fci(self):
+        return self
```

Comment:
Are these three not inherited from the spin-restricted class?

Reply:
Yep